corehq/motech/value_source.py

```diff
@@ -145,8 +145,8 @@ def _get_commcare_value(self, case_trigger_info):
         1
         >>> CaseProperty(case_property="bar")._get_commcare_value(info)
         2
-        >>> type(CaseProperty(case_property="baz")._get_commcare_value(info))
-        <type 'NoneType'>
+        >>> type(CaseProperty(case_property="baz")._get_commcare_value(info)) == type(None)
```

Comment:
Is there a reason it couldn't be changed to 

`CaseProperty(case_property="baz")._get_commcare_value(info) is None`

Reply:
brilliant: https://github.com/dimagi/commcare-hq/pull/23220/commits/a2e3076803993b6040bd6a4e29fc00c43d44f0c8